Plugin need Java 11 and does not run with Java 8

When I run with Java 8, I get an `NoSuchMethodError`:

```
Caused by: java.lang.NoSuchMethodError: java.util.List.toArray(Ljava/util/function/IntFunction;)[Ljava/lang/Object;
	at com.nocwriter.runsql.gradle.ScriptsReader.readScriptFile(ScriptsReader.java:65)
	at com.nocwriter.runsql.gradle.ScriptsReader.lambda$fetchScripts$0(ScriptsReader.java:38)
	at com.nocwriter.runsql.gradle.ScriptsReader.fetchScripts(ScriptsReader.java:36)
	at com.nocwriter.runsql.gradle.RunSQLTask.execute(RunSQLTask.java:49)
```

The ScriptsReader is used a method `toArray()` introduced in Java 11.

See https://github.com/NocWriter/runsql-gradle-plugin/blob/e1fcd574adfca1980716c052943453777c0b59b9/src/main/java/com/nocwriter/runsql/gradle/ScriptsReader.java#L65

See https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/util/Collection.html#toArray(java.util.function.IntFunction)

--- a/src/main/java/com/nocwriter/runsql/jdbc/JdbcUtils.java
+++ b/src/main/java/com/nocwriter/runsql/jdbc/JdbcUtils.java
@@ -11,6 +11,7 @@ import java.sql.Connection;
 import java.sql.Driver;
 import java.sql.DriverManager;
 import java.sql.SQLException;
+import java.util.Collections;
 import java.util.LinkedList;
 import java.util.List;
 
@@ -120,7 +121,7 @@ public class JdbcUtils {
         StringBuilder buf = new StringBuilder();
         buf.append("Detected JDBC drivers:\n");
         List<String> availableDrivers = new LinkedList<>();
-        DriverManager.drivers().forEach(driver -> {
+        Collections.list(DriverManager.getDrivers()).forEach(driver -> {
             String location = driver.getClass().getResource("/" + driver.getClass().getName().replace('.', '/') + ".class").toString();
             String[] locationParts = location.split("!");
             String[] pathParts = locationParts[0].split("/");

--- a/src/main/java/com/nocwriter/runsql/script/ScriptParser.java
+++ b/src/main/java/com/nocwriter/runsql/script/ScriptParser.java
@@ -68,7 +68,7 @@ public class ScriptParser {
             lineNumber++;
 
             // Skip empty or comment lines.
-            if (line.isBlank() || line.trim().startsWith("--")) {
+            if (line.trim().isEmpty() || line.trim().startsWith("--")) {
                 continue;
             }
 

